src/LiveComponent/tests/Fixture/Component/Component6.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LiveComponent\Tests\Fixture\Component;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\RangeType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\ComponentWithFormTrait;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+/**
+ * @author Jakub Caban <kuba.iluvatar@gmail.com>
+ */
+#[AsLiveComponent('component6')]
+class Component6 extends AbstractController
```

Comment:
Conflicts conflicts :D
https://github.com/symfony/ux/pull/218/files#diff-26b9f4a13fdb743a0f3fb04b6b3606c1c6e3ba47c4d92e58fd0a25aa5c8df7f7

Reply:
First merge wins!

Reply:
Haha, I noticed that too when reviewing. Maybe we should be a bit more creative with our test class name 😆

Reply:
I think I will just rename it to `FormComponent1` or something along those lines. Not really creative, but at least it will separate concerns a bit :smile: